config/rollup/rollup.config.js

```diff
@@ -86,12 +86,21 @@ function configure(pkg, env, target) {
     builtins(),
 
     // Use Babel to transpile the result, limiting it to the source code.
-    babel({
-      runtimeHelpers: true,
-      include: [`packages/${pkg.name}/src/**`],
-      extensions: ['.js', '.ts'],
-      configFile: './config/babel/babel.config.cjs',
-    }),
```

Comment:
Instead of splitting the `babel.config.cjs` file in two. Could we inline it here in the Rollup config? Since that's the only place we're using it. That way we can have the smallest possible footprint and most clarity on specifically what `isUmd` enables?

I'm imagining something like:

```js
    babel({
      runtimeHelpers: true,
      include: [`packages/${pkg.name}/src/**`],
      extensions: ['.js', '.ts'],
      presets: [
        '@babel/preset-typescript',
        ['@babel/preset-env', { 
          exclude: isUmd ? ... : ... ,
          modules: false }
        ],
        '@babel/preset-react',
      ],
      plugins: [
        ['@babel/plugin-transform-runtime', isUmd ? {...} : {}],
        '@babel/plugin-proposal-class-properties',
      ],
    })
```

And then we can get rid of `babel.config.js` altogether!

I'm just thinking about trying to figure out what necessitated the multiple files in the future and being very confused.

Thank you!

Reply:
Good call - I'll take a look tonight. Had no idea the rollup plugin handled this and it's actually a decent improvement I can add to a couple of my other projects so thanks for pointing this out.